synapse/federation/transport/server/federation.py

```diff
@@ -435,17 +434,15 @@ async def on_PUT(
         # TODO(paul): assert that event_id parsed from path actually
         #   match those given in content
 
-        partial_state = False
-        if self._msc3706_enabled:
-            # The stable query parameter wins, if it disagrees with the unstable
-            # parameter for some reason.
-            stable_param = parse_boolean_from_args(query, "omit_members", default=None)
-            if stable_param is not None:
-                partial_state = stable_param
-            else:
-                partial_state = parse_boolean_from_args(
-                    query, "org.matrix.msc3706.partial_state", default=False
-                )
+        # The stable query parameter wins, if it disagrees with the unstable
+        # parameter for some reason.
+        stable_param = parse_boolean_from_args(query, "omit_members", default=None)
+        if stable_param is not None:
+            partial_state = stable_param
+        else:
+            partial_state = parse_boolean_from_args(
+                query, "org.matrix.msc3706.partial_state", default=False
+            )
```

Comment:
Hmm. We were hoping to be fairly cavalier here, so that we can get as much testing as we can when the next RC goes out. Will ponder this more carefully next week

Reply:
See d4a4cd3.

Reply:
I don't feel super strongly, but it is feedback that has come up in the past about stabilizing features... then you accidentally end up needing to support both for a while. 😢